pkg/networkservice/common/excludedprefixes/utils.go

```diff
@@ -18,7 +18,8 @@
 
 package excludedprefixes
 
-func removeDuplicates(elements []string) []string {
+// RemoveDuplicates - removes duplicate strings from a slice
+func RemoveDuplicates(elements []string) []string {
```

Comment:
Why is this public?

Reply:
This is used in tests which is in separate package. Shall we move this to `tools`?

Reply:
Removed